packages/i18n/src/ja.ts

```diff
@@ -0,0 +1,66 @@
+import { Language } from './types';
+
+// prettier-ignore
+const language: Language = {
+  code:             'ja',
+  schema:           (issue) => `無効な型: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
```

Comment:
```suggestion
  schema:           (issue) => `無効な型: ${issue.expected} を期待したが、 ${issue.received} を受け取りました`,
```

`受け取った` is a mechanical way of saying it, so how about saying `受け取りました`?

Reply:
Since the English message was short, I was trying to shorten my message as well.
https://github.com/fabian-hiller/valibot/blob/cbd1b25decd7f4f615510cf5dcdc2ec75d73c83f/packages/i18n/src/en.ts#L6

But I agree, I'm going to modify them.

Reply:
I changed them by 85b765b.

Reply:
`受け取りました`だと途中の`を期待したが`が堅苦しく感じますね。
なので、`期待しましたが`としても良いかなと思いました。 @roottool

Reply:
I changed from `を期待したが` to `期待しましたが` by 127cee2. BTW, How do you think [this comment](https://github.com/fabian-hiller/valibot/pull/431#issuecomment-1937021661)?

Reply:
I think the subject of these messages is Valibot, so it should be fine to keep them consistent in the active voice.
However, I don't care too much :)

Reply:
Thank you for your opinion! I understood, I decide not to adopt it.